docs/docs/dev_docs/contracts/portals/main.md

```diff
@@ -180,7 +189,7 @@ Designated callers are enforced at the contract level for contracts that are not
 
 - Non-native token (asset bridged from L1)
   - [Portal contract](https://github.com/AztecProtocol/aztec-packages/blob/master/l1-contracts/test/portals/TokenPortal.sol)
-  - [Aztec contract](https://github.com/AztecProtocol/aztec-packages/blob/master/yarn-project/noir-contracts/src/contracts/non_native_token_contract/src/main.nr)
+  - [Aztec Token Bridge contract](https://github.com/AztecProtocol/aztec-packages/blob/master/yarn-project/noir-contracts/src/contracts/token_bridge_contract/src/main.nr)
 - Uniswap (swap tokens on L1)
```

Comment:
Are the uniswap in here still functional otherwise I think we should remove it from the links. Having a broken example seems non-ideal.